presto-hive/src/test/java/com/facebook/presto/hive/metastore/TestCachingHiveMetastore.java

```diff
@@ -148,35 +156,28 @@ public void testGetTable()
         assertEquals(mockClient.getAccessCount(), 2);
     }
 
-    public void testInvalidDbGetTable()
```

Comment:
Instead of removing it, can we try to still test this with the updated changes? Do you see an issue while adding this test?

Reply:
Added this test back using another table variable - `RANDOM_TABLE`. This was the easiest way out without breaking anything and for keeping this test case in place. Let me know if you think otherwise.